apps/documentation/src/routes/documentation/contributing.mdx

```diff
@@ -1,15 +1,14 @@
-import { Head } from 'tuono'
+import MetaTags from '../../components/meta-tags'
 
-<Head>
-  <title>Tuono - Contributing</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/contributing" />
-</Head>
+<MetaTags
+  title="Tuono - Contributing"
+  canonical="https://tuono.dev/documentation/contributing"
+  description="The project is massive - if you like it, do consider contributing!"
+/>
 
 import Breadcrumbs, { Element } from '../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="✨ Contributing" />
-</Breadcrumbs>
+<Breadcrumbs breadcrumbs={[{ label: '✨ Contributing' }]} />
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space after the opening parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `{`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChAxNTg5NGUzOWRiMzMzZjQ2EgxMYW5ndWFnZVRvb2w= -->